src/Context.re

```diff
@@ -21,3 +21,19 @@ module SectionSlugContext = {
   let makeProps = ReactExt.makeProps;
   let useSection = () => React.useContext(context);
 };
+
+module LanguageContext = {
+  type t =
+    | English
+    | Yiddish;
+  let stringOfLanguage = t => {
+    switch (t) {
+    | English => "en"
+    | Yiddish => "ji"
```

Comment:
do you need Russian here? Is this dead code now?

Reply:
Deleted this now and it's replaced with Russian.